lib/entities/quote/DutchQuote.ts

```diff
@@ -364,7 +364,7 @@ export class DutchQuote implements IQuote {
   }
 
   public get amountIn(): BigNumber {
-    return this.amountInStart;
+    return this.amountInStart.sub(this.portionAmountInStart);
```

Comment:
maybe we want to add comments; so amountInStart is the ones with fees included right? thats why we are substracting it!

Reply:
yep yep

Reply:
so now `amountIn` and `amountInGasAdjusted` is the same thing. Shouldnt we technically substract the cost of gas denominated in tokenIn?